src/main/java/io/xpring/ilp/IlpClient.java

```diff
@@ -0,0 +1,101 @@
+    protected IlpClient(String grpcUrl) {
+        this.decoratedClient = new DefaultIlpClient(grpcUrl);
+    }
+
+    /**
+     * Create an account on the connector.  Because no settings or auth were passed in, an account
+     * with default settings, a generated account ID, and a generated SIMPLE auth token will be generated for you
+     *
+     * @return A {@link CreateAccountResponse} containing the account settings that were created on the connector
+     * @throws XpringException If account creation failed.
+     */
+    public CreateAccountResponse createAccount() throws XpringException {
+        return decoratedClient.createAccount();
+    }
+
+    /**
+     * Create an account on the connector.
+     *
+     *
+     * @param createAccountRequest: A request object with specified account details. Note that {@link CreateAccountRequest#assetScale}
+     *                           and {@link CreateAccountRequest#assetCode} MUST be specified, as guaranteed by {@link CreateAccountRequest#builder(String, Integer)}
+     * @return A {@link CreateAccountResponse} containing the account settings that were created on the connector
+     * @throws XpringException If the given inputs were invalid or account creation failed.
+     */
+    public CreateAccountResponse createAccount(CreateAccountRequest createAccountRequest, Optional<String> bearerToken) throws XpringException {
+        return decoratedClient.createAccount(createAccountRequest, bearerToken);
+    }
+
+    /**
+     * Gets account details for the given account ID
+     *
+     * @param accountId
+     * @param bearerToken
+     * @return
+     * @throws XpringException
+     */
+    public GetAccountResponse getAccount(String accountId, String bearerToken) throws XpringException {
+        return decoratedClient.getAccount(accountId, bearerToken);
+    }
+
+    /**
+     * Get the balance of the specified account on the connector.
+     *
+     * @param accountId The account ID to get the balance for.
+     * @param bearerToken Authentication bearer token. TODO: Probably change from string to some wrapped JWT class
+     * @return A {@link BigInteger} with the number of drops in this account.
+     * @throws XpringException If the given inputs were invalid.
+     */
+    public GetBalanceResponse getBalance(final String accountId, final String bearerToken) throws XpringException {
+        return decoratedClient.getBalance(accountId, bearerToken);
+    }
+
+    /**
+     * Send a payment from the given accountId to the destinationPaymentPointer payment pointer
+     * @param destinationPaymentPointer : payment pointer of the receiver
+     * @param amount : Amount to send
+     * @param accountId : accountId of the sender
+     * @param bearerToken : auth token of the sender
+     * @return
+     * @throws XpringException
+     */
+    public SendPaymentResponse sendPayment(final String destinationPaymentPointer,
+                                           final long amount,
```

Comment:
Why type this as a long rather than an arbitrary precision `BigInt`?

Reply:
ILP only supports unsigned longs per the protocol. We should use `UnsignedLong` here though, if possible. @nkramer44  would that work?

Reply:
We can use `UnsignedLong` to align with ILP.  I typed it as `long` originally because `SendPaymentRequest` requires a `long`.